ext/droption/droption.h

```diff
@@ -46,6 +46,8 @@
 #define TESTALL(mask, var) (((mask) & (var)) == (mask))
 #define TESTANY(mask, var) (((mask) & (var)) != 0)
 
+#define DROPTION_DEFAULT_VALUE_SEP " "
+
 // XXX: some clients want further distinctions, such as options passed to
 // post-processing components, internal (i.e., undocumented) options, etc.
 /**
```

Comment:
Github won't let me comment on line 72 but please update the docs there: currently it says `(separating each appended value with a space)`.  Please change it to something like `(separating each appended value with a space by default or with a user-specified accumulated value separator if it was supplied to the droption_t constructor)`.

Reply:
Updated!